listener.go

```diff
@@ -152,13 +153,13 @@ type listener struct {
 func (l *listener) WrapExpiration(days int) net.Listener {
 	ctx, cancel := context.WithCancel(context.Background())
 	go func() {
-		time.Sleep(5 * time.Minute)
+		time.Sleep(30 * time.Second)
```

Comment:
Can you include a comment as to why we're sleeping here and the significance of 30 seconds?

Reply:
I am not sure what the original intent was in sleeping for 5 minutes at startup before checking certificate renewal. I have not seen any particular reason for it based on how dynamiclistener is used in k3s and rancher. @ibuildthecloud or @galal-hussein may have some context?

In practice, it makes it harder to test the fix.

Reply:
Seems like a decent balance.